docs.md

```diff
@@ -1,9 +1,251 @@
-## Configuration 
+# Document for Developers
+___
+### Libraries used:
```

Comment:
Not necessary. This should just be document about how can one clone the project and setup in local and test and deploy.
1. git clone etc
2. cd into some diretory 
3. pip install req
4. how to check if API's are working meaning opening swagger and uploading a base64 string
5. how to make changes and deploy